### ISSUE TITLE
Error in UseMethod("html_table") 

### ISSUE THREAD

**Issue by aniruhil** [kent-orr/oshaR#2]
Neat! Let me be the first user to file an issue! 

```
library(oshaR)

osha_search(
   std_query = "Radiation") -> rad
```

> Error in UseMethod("html_table") : 
 >  no applicable method for 'html_table' applied to an object of class "xml_missing"

```
sessionInfo(): 
```
<details>
R version 4.0.4 (2021-02-15)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur 10.16

Matrix products: default
LAPACK: /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] dplyr_1.0.4 oshaR_0.1.0

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.6        lubridate_1.7.9.2 assertthat_0.2.1  crayon_1.4.1     
 [5] R6_2.5.0          DBI_1.1.1         lifecycle_1.0.0   magrittr_2.0.1   
 [9] httr_1.4.2        pillar_1.4.7      rlang_0.4.10      stringi_1.5.3    
[13] curl_4.3          xml2_1.3.2        vctrs_0.3.6       generics_0.1.0   
[17] ellipsis_0.3.1    tools_4.0.4       stringr_1.4.0     glue_1.4.2       
[21] purrr_0.3.4       yaml_2.2.1        compiler_4.0.4    pkgconfig_2.0.3  
[25] rvest_0.3.6       tidyselect_1.1.0  tibble_3.0.6   
</details>

**Comment by kent-orr** [comment]
The search results are a literal data scrape from the html table returned. If there are no search results, the GET request does not return an html page with the corresponding node the output dataframe is scraped from. Implemented a tryCatch that provides a more informative  error message.

**Comment by aniruhil** [comment]
Yes but the OSHA website returns 2 results for `Radiation` ... so there should be a non-empty html table.

**Comment by kent-orr** [comment]
Looks like the responses from the site itself are outside the default start_date supplied to the query `Sys.Date() - 365`. Try `osha_search('radiation', start_date = '2016-01-01')`

```
  # inspection citation   issuance report_id category sic                      establishment_name ...
1 1    1345410     1001 02/13/2019    420600       NA  NA        Nestle Waters North America Inc. ...
2 2    1135993     1001 05/16/2016    420600       NA  NA Allpoints Warehousing Equipment Company ...
```
